setup.py

```diff
@@ -12,8 +13,10 @@
     version = ''
 try:
     # obtain long description from README and CHANGES
-    README = open(os.path.join(here, 'README.rst')).read()
-    CHANGES = open(os.path.join(here, 'CHANGES.rst')).read()
+    with open(os.path.join(here, 'README.rst'), 'r', encoding='utf-8') as f:
+        README = f.read()
+    with open(os.path.join(here, 'CHANGES.rst'), 'r', encoding='utf-8') as f:
+        CHANGES = f.read()
```

Comment:
I should think that pip will handle `unicode` without a problem. I looked through pip and it seems they use a `unicode` string for the example, so it shouldn't cause any issues.
They also use `codecs.open()`, though I don't see what the difference is from `io.open()`